morphocluster/api.py

```diff
@@ -232,10 +232,13 @@ def get_file(path):
     )
 
 
-@api.route("/files/<string:path>", methods=["GET"])
-def get_files(path):
-    
-    directory_path = app.config["FILES_DIR"] 
+@api.route("/files/<path:path>", methods=["GET"])
+@api.route("/files/", methods=["GET"])
+def get_direntry(path=""):
+    if path != "":
+        directory_path = app.config["FILES_DIR"]+"/"+path
```

Comment:
Use `os.path.join`. This also takes care of an empty `path` so you don't need a `if`-conditional.